packages/loot-core/package.json

```diff
@@ -49,29 +48,24 @@
     "cross-env": "^7.0.3",
     "date-fns": "2.0.0-alpha.27",
     "eslint": "8.35.0",
-    "esm": "^3.0.82",
     "fake-indexeddb": "^3.1.3",
     "fast-check": "2.13.0",
-    "fast-glob": "^2.2.0",
-    "jest": "^28.1.0",
+    "jest": "^27.0.0",
```

Comment:
Did you intentionally downgrade `jest`?

Reply:
Yes, the latest `react-scripts` is on Jest 27 (and we pull in `jest-environment-jsdom` implicitly from that), so we need to use Jest 27 here too.